<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdlQtRBGClass</class>
 <widget class="QMainWindow" name="SdlQtRBGClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SdlQtRBG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="video1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(16, 28, 255);</string>
    </property>
    <property name="text">
     <string>video</string>
    </property>
   </widget>
   <widget class="QLabel" name="video2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pix1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>YUV420P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGBA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BGRA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB24</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="width1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>400</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="height1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="set_fps1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="height2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="open2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="width2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>400</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="set_fps2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QComboBox" name="pix2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>YUV420P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGBA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BGRA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB24</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="fps1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fps2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sdlqtrbg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>open1</sender>
   <signal>clicked()</signal>
   <receiver>SdlQtRBGClass</receiver>
   <slot>Open1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>637</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open2</sender>
   <signal>clicked()</signal>
   <receiver>SdlQtRBGClass</receiver>
   <slot>Open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open1()</slot>
  <slot>Open2()</slot>
 </slots>
</ui>
